controllers/nc-news4.controller.js

```diff
@@ -0,0 +1,12 @@
+const getArticlesById = require("../models/nc-news4.model");
+
+const getArticles = (req, res) => {
+  const { article_id } = req.params;
+  getArticlesById(article_id)
+    .then((articles) => {
+      res.status(200).send({ articles });
```

Comment:
There's only one article being sent back, so it's more appropriate to de-pluralise this :)